sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -3831,49 +3851,83 @@ SUBROUTINE CLDRAD
       ENDIF
 
 ! Instantaneous clear-sky downwelling LW at the surface
-      IF (IGET(744)>0) THEN
+      IF (IGET(764)>0) THEN
```

Comment:
@EricJames-NOAA Thanks for catching the issue of instant clear-sky downwelling/upwelling long-wave flux having no UPP IDs. Could you add these two definitions in post_avblflds.xml as:
```
    <param>
       <post_avblfldidx>764</post_avblfldidx>
       <shortname>CSDLF_ON_SURFACE</shortname>
       <pname>CSDLF</pname>
       <fixed_sfc1_type>surface</fixed_sfc1_type>
       <scale>3.0</scale>
    </param>
  
       <param>
       <post_avblfldidx>765</post_avblfldidx>
       <shortname>CSULF_ON_SURFACE</shortname>
       <pname>CSULF</pname>
       <fixed_sfc1_type>surface</fixed_sfc1_type>
       <scale>3.0</scale>
    </param>
``` 
We might use them in the future. Thanks!